build/Build.cs

```diff
@@ -26,20 +23,20 @@ class Build : NukeBuild
     [Parameter("Branch name for OctoVersion to use to calculate the version number. Can be set via the environment variable " + CiBranchNameEnvVariable + ".", Name = CiBranchNameEnvVariable)]
     readonly string OctoVersionBranch;
 
-    [Parameter] readonly int? OctoVersionFullSemVer;
+    [Parameter] readonly string? OctoVersionFullSemVer;
```

Comment:
This one caught me by surprise. After upgrading, [Nuke started to complain about this](https://github.com/OctopusDeploy/Versioning/actions/runs/6965352998/job/18953903753#step:5:30). Weird that it wasn't doing it before.